rootfs/usr/share/www/static/styles.css

```diff
@@ -0,0 +1,147 @@
+@font-face {
+  font-family: "Roboto";
+  src: local("Roboto Regular"), local("Roboto-Regular"),
+    url(/landingpage/static/Roboto-Regular.woff2) format("woff2");
+  font-weight: 400;
+  font-style: normal;
+}
+
+html {
+  color: var(--primary-text-color, #212121);
+  background-color: #0277bd !important;
+}
+
+body {
+  font-family: Roboto, Helvetica, Arial, sans-serif;
+  padding-top: 10%;
```

Comment:
bit weird that we have a padding-top here and `.content` has a margin-top: 64px. Should we drop this one to align it with onboarding?